runtime/doc/builtin.txt

```diff
@@ -2021,6 +2021,8 @@ feedkeys({string} [, {mode}])				*feedkeys()*
 		't'	Handle keys as if typed; otherwise they are handled as
 			if coming from a mapping.  This matters for undo,
 			opening folds, etc.
+		'L'	Lowlevel input like |nvim_input()|.  Other flags are
```

Comment:
what's the difference compared with calling `nvim_input()` directly?

Reply:
Would be good to make that clear and explicit in the docs (I couldn't tell what this does at all...)